integrations/SnapPixel/util.js

```diff
@@ -208,4 +231,24 @@ const ecommEventPayload = (event, message) => {
   return payload;
 };
 
-export { sendEvent, ecommEventPayload, eventPayload };
+const getUserEmailAndPhone = (hashMethod, userEmail, userPhoneNumber) => {
```

Comment:
Please add documentation for this function

Reply:
Updated